--- mednafen/config.rpath.orig	2006-12-22 05:30:02.000000000 +0000
+++ mednafen/config.rpath	2007-01-03 19:37:59.000000000 +0000
@@ -153,7 +153,7 @@
   # here allows them to be overridden if necessary.
   # Unlike libtool, we use -rpath here, not --rpath, since the documented
   # option of GNU ld is called -rpath, not --rpath.
-  hardcode_libdir_flag_spec='${wl}-rpath ${wl}$libdir'
+  hardcode_libdir_flag_spec=''
   case "$host_os" in
     aix3* | aix4* | aix5*)
       # On AIX/PPC, the GNU linker is very broken
@@ -191,7 +191,7 @@
       ;;
     interix3*)
       hardcode_direct=no
-      hardcode_libdir_flag_spec='${wl}-rpath,$libdir'
+      hardcode_libdir_flag_spec=''
       ;;
     linux*)
       if $LD --help 2>&1 | grep ': supported targets:.* elf' > /dev/null; then
@@ -218,7 +218,7 @@
           ;;
         *)
           if $LD --help 2>&1 | grep ': supported targets:.* elf' > /dev/null; then
-            hardcode_libdir_flag_spec='`test -z "$SCOABSPATH" && echo ${wl}-rpath,$libdir`'
+            hardcode_libdir_flag_spec='`test -z "$SCOABSPATH"`'
           else
             ld_shlibs=no
           fi
@@ -399,7 +399,7 @@
       fi
       ;;
     irix5* | irix6* | nonstopux*)
-      hardcode_libdir_flag_spec='${wl}-rpath ${wl}$libdir'
+      hardcode_libdir_flag_spec=''
       hardcode_libdir_separator=:
       ;;
     netbsd*)
@@ -408,20 +408,20 @@
       ;;
     newsos6)
       hardcode_direct=yes
-      hardcode_libdir_flag_spec='${wl}-rpath ${wl}$libdir'
+      hardcode_libdir_flag_spec=''
       hardcode_libdir_separator=:
       ;;
     openbsd*)
       hardcode_direct=yes
       if test -z "`echo __ELF__ | $CC -E - | grep __ELF__`" || test "$host_os-$host_cpu" = "openbsd2.8-powerpc"; then
-        hardcode_libdir_flag_spec='${wl}-rpath,$libdir'
+        hardcode_libdir_flag_spec=''
       else
         case "$host_os" in
           openbsd[01].* | openbsd2.[0-7] | openbsd2.[0-7].*)
             hardcode_libdir_flag_spec='-R$libdir'
             ;;
           *)
-            hardcode_libdir_flag_spec='${wl}-rpath,$libdir'
+            hardcode_libdir_flag_spec=''
             ;;
         esac
       fi
@@ -431,15 +431,15 @@
       hardcode_minus_L=yes
       ;;
     osf3*)
-      hardcode_libdir_flag_spec='${wl}-rpath ${wl}$libdir'
+      hardcode_libdir_flag_spec=''
       hardcode_libdir_separator=:
       ;;
     osf4* | osf5*)
       if test "$GCC" = yes; then
-        hardcode_libdir_flag_spec='${wl}-rpath ${wl}$libdir'
+        hardcode_libdir_flag_spec=''
       else
         # Both cc and cxx compiler support -rpath directly
-        hardcode_libdir_flag_spec='-rpath $libdir'
+        hardcode_libdir_flag_spec=''
       fi
       hardcode_libdir_separator=:
       ;;
